File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II/0 - Revisao/3 - Entrada e Saída/inputOutput.py
print() #Saída Padrão
input() #Entrada Padrão

nome = input("Digite seu primeiro nome: ")
idade = int(input("Digite sua idade: ")) #Faz a leitura em String e converte para numérico
altura = float(input("Digite sua altura: "))

print("{0} tem {1} anos de idade e mede {2}m.".format(nome, idade, altura))
print(nome, " tem ", idade, " anos de idade e mede ", altura, "m.")

File: importado-do-antigo/ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II/teo/U0-nivelamento/0-Intro_Lang_Prog/aula3/inputOutput.cs
using System;

class HelloWorld{
    public static void Main(string[] args){
        
        Console.ReadLine(); //Entrada
        //Sempre retorna um String, se for ler números, tem que converser com Parse()

        Console.WriteLine ("Hello World!"); //Saída + Pula 1 Linha
        Console.Write("Hello World!"); //Saída Padrão

        //Exemplo
        string nome;
        int idade;
        float altura;

        Console.Write("Digite seu primeiro nome: ");
        x = Console.ReadLine();

        Console.Write("Digite sua Idade: ");
        idade = int.Parse(Console.ReadLine());

        Console.Write("Digite sua altura: ");
        altura = float.Parse(Console.ReadLine());

        Console.WriteLine("\n" + nome + " tem " + idade + " anos de idade e mede " + altura + "m.");
    }
}